cmake/ExternalProjects.cmake

```diff
@@ -45,12 +45,12 @@ set_target_properties(pistache
 )
 
 # RapidJSON
+set(RAPIDJSON_BUILD_DOC OFF CACHE INTERNAL "") 
+set(RAPIDJSON_BUILD_EXAMPLES OFF CACHE INTERNAL "") 
+set(RAPIDJSON_BUILD_TESTS OFF CACHE INTERNAL "") 
```

Comment:
This seems to actually work vs. the previous version which apparently didn't pass the args on